ciao-controller/openstack_compute.go

```diff
@@ -569,6 +569,10 @@ func buildFlavorDetails(workload types.Workload) (compute.FlavorDetails, error)
 	details.Disk = workload.ImageID
 	details.Name = workload.Description
 
+	if details.Disk == "" {
+		details.Disk = workload.Storage[0].SourceID
```

Comment:
@rbradford ok got it, so, is this ok to have empty value in ``ciao-cli`` output when we use ciao-image provided image?